### ISSUE TITLE
Sort by last aired

### ISSUE THREAD

**Issue by TheSolution444** [rebelonion/Dantotsu#220]
Could you add the option to sort list by last aired?

**Comment by AbandonedCart** [comment]
Doesn't https://github.com/rebelonion/Dantotsu/pull/272 do this?

**Comment by TheSolution444** [comment]
> Doesn't https://github.com/rebelonion/Dantotsu/pull/272 do this?

No this doesn't include the option to sort by last episode air date

**Comment by JohnVictoryz** [comment]
what do you mean by this do you mean when trying to watch an anime like show the episodes it descending order so basically by last aired or something else

**Comment by TheSolution444** [comment]
> what do you mean by this do you mean when trying to watch an anime like show the episodes it descending order so basically by last aired or something else 

No. I'm referring to the user's list which is accessed from the homepage. ![Screenshot_20240414-122848.png](https://github.com/rebelonion/Dantotsu/assets/161716107/d387b422-fa3c-4478-92f4-1a719ac0ac4a)

The ability to sort by recently updated similar to how it is done in the "Recently Updated" category on the "Anime" page is the feature I'm looking for.

**Comment by JohnVictoryz** [comment]
> > what do you mean by this do you mean when trying to watch an anime like show the episodes it descending order so basically by last aired or something else 
> 
> No. I'm referring to the user's list which is accessed from the homepage. ![Screenshot_20240414-122848.png](https://github.com/rebelonion/Dantotsu/assets/161716107/d387b422-fa3c-4478-92f4-1a719ac0ac4a)
> 
> The ability to sort by recently updated similar to how it is done in the "Recently Updated" category on the "Anime" page is the feature I'm looking for.

Ok now I understand what you mean I will look into it it would be quite easy but it will take some time because I am not that familiar with this codebase

**Comment by JohnVictoryz** [comment]
a bit of an update that brings bad news unfortunately the way that the list sort works is by sending a request to anilist with the type of sort they want and then just anilist does the work and returns a list and in anilist it doesnt have a sort option like this from what i know

**Comment by TheSolution444** [comment]
> a bit of an update that brings bad news unfortunately the way that the list sort works is by sending a request to anilist with the type of sort they want and then just anilist does the work and returns a list and in anilist it doesnt have a sort option like this from what i know

Oh that's sad
